Html.playground/Contents.swift

```diff
@@ -32,31 +32,30 @@ code {
 """
 
 /// A document built in the HTML DSL.
-let doc = document(
-  doctype,
-  html(
-    head(
-      style(unsafe: stylesheet)
+let doc: Node = .document(
+  .html(
+    .head(
+      .style(safe: stylesheet)
```

Comment:
One weird concession was without making `safe` vs. `unsafe` explicit, I'd get ambiguous errors that required `as StaticString` even on string literals. Was strange and maybe worth filing a bug for!

Reply:
oh dang strange